bricks/template/__brick__/{{project_name.snakeCase()}}/README.md

```diff
@@ -11,6 +11,8 @@
 
 ### Setup
 
+- Dependency Injection (DI), State Management, and Navigating with [get_it](https://pub.dev/packages/get_it), [flutter_riverpod](https://pub.dev/packages/flutter_riverpod), and [go_router](https://pub.dev/packages/go_router).
+
```

Comment:
As mentioned [here](https://github.com/nimblehq/flutter-templates/pull/244#discussion_r1274467888), this content should not be put here in the generated project, but update this line from the root's README https://github.com/nimblehq/flutter-templates/blob/ab5559ccd8c79dd819f1f538303d4a4f54b92148/README.md?plain=1#L10